experiments/tf_trainer/tf_hub_classifier/model.py

```diff
@@ -24,7 +24,7 @@
 # This would normally just be a multi_integer, but we use string due to
 # constraints with ML Engine hyperparameter tuning.
 tf.app.flags.DEFINE_string(
-    'dense_units', '512,128,64',
+    'dense_units', '1024,1024,512',
```

Comment:
Why the same number twice?

Reply:
Multiple layers, all the same size.

Reply:
Ah, thanks. Then maybe add to the description to say: "number of hidden units in in the dense layers." ?